wrapper/src/test/java/integration/util/AuroraTestUtility.java

```diff
@@ -1415,6 +1415,7 @@ public List<String> getEngineVersions(String engine) {
 
   public String getLatestVersion(String engine) {
     return getEngineVersions(engine).stream()
+        .filter(version -> !version.contains("limitless"))
```

Comment:
Do we want to filter out other versions that are suffixed in case another non-regular version is added in the future? Or would we prefer to deal with that later? If we decide on the former I will change to the code below. Note that currently the only version that contains a "-", and the only non-regular version, is "16.4-limitless"
```suggestion
        .filter(version -> !version.contains("limitless"))
```

Reply:
I'd prefer a strict condition with "-limitless"